mmv1/third_party/terraform/services/storage/resource_storage_bucket.go.erb

```diff
@@ -1640,6 +1640,14 @@ func resourceGCSBucketLifecycleRuleConditionHash(v interface{}) int {
 		buf.WriteString(fmt.Sprintf("%s-", v.(string)))
 	}
 
+	if v, ok := m["custom_time_before"]; ok {
+		buf.WriteString(fmt.Sprintf("%s", v.(string)))
```

Comment:
I think the hyphen delimiter is important here (at least generally), to mark the start and end of a field. Am I missing something?

Reply:
Nope. I think, I missed putting hyphen there. Will update the PR.

Reply:
Updated PR.